todo.sh

```diff
@@ -6,57 +6,69 @@
 # like so $lib_dir/someProjectScript.sh
 this_path=${BASH_SOURCE[0]}
 lib_dir="$(dirname "$this_path")"
-
 export todo_task_file="${TODO_TASK_FILE:-$lib_dir/.todo}"
+export padding="       "
 todo_header_file="${TODO_HEADER_FILE:-$lib_dir/table_head.sh}"
 no_local=false # Skip reading of local .todo file
 
-export padding="       "
+# Don't know if there is a better way to do this??
```

Comment:
You can do this in a loop several ways like

```sh
this_path=${BASH_SOURCE[0]}
this_dir="$(dirname "$this_path")"
this_file="$(basename "$this_path")"

for src in *.sh; do
    if [ "$src" = "$this_file" ]; then
        continue
    fi

    . "$this_dir/$src"
done
```

Or

```sh
this_path=${BASH_SOURCE[0]}
this_dir="$(dirname "$this_path")"
this_file="$(basename "$this_path")"

for src in $(find . -maxdepth 1 -mindepth 1 -name "*.sh" -a -not -name "$this_file"); do
    . "$this_dir/$src"
done
```
Or similar.

Reply:
Ok, I think  I managed to get this working